Sage/carbon
# docs(toast): fix bug with stacked example

fixes #3336

### Proposed behaviour

![Toast_stacked_fix](https://user-images.githubusercontent.com/56251247/98133871-06afd580-1eb6-11eb-9b11-53a8d9ddd327.gif)


### Current behaviour

![toast_stacked_bug](https://user-images.githubusercontent.com/56251247/98236430-3d8bf700-1f5b-11eb-9680-d083481ee2f6.gif)

### Checklist



### Additional context
N/A

### Testing instructions
- `git checkout FE-3283-fix_toast_story`
- `npm start` to start Storybook
- Navigate to Toast -> stacked 
- Example should now work as expected.
